redhat-developer/app-services-cli
# feat: added de-register command

Closes # 

### Verification Steps

1. Have a working and registerd cluster with a kafka `rhoas dedicated register-cluster`
2. Run deregister `rhoas dedicated deregister-cluster`
3. All kafkas should delete first then deregistered from kfm then removing addons

### Type of change
